code/HashMaps/CountDistinctArrayElements.java

```diff
@@ -0,0 +1,25 @@
+import java.util.*;
+class Example1
+{
+	public static void main(String[] args)
+	{
+		Scanner sc=new Scanner(System.in);
+		int n=sc.nextInt();
+		int[] arr=new int[n];
+		for(int i=0;i<n;i++)
```

Comment:
Add a space between the for and the (,and the ) and the {. Do this for both loops.